lib/collections.js

```diff
@@ -1,2 +1,4 @@
-
+Gists = new Mongo.Collection("gists")
+Files = new Mongo.Collection("files")
 Documents = new Mongo.Collection("documents")
+EditingUsers = new Mongo.Collection("editingUsers")
```

Comment:
@calvintan separated Gists and Files collection due Github's use of '.' in filenames as object keys